modules/iam-circleci-oidc-provider/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "create" {
+  description = "Controls if resources should be created (affects all resources)"
+  type        = bool
+  default     = true
+  nullable    = false
+}
+
+variable "tags" {
+  description = "A map of tags to add to the resources created"
+  type        = map(any)
+  default     = {}
+  nullable    = false
+}
+
+variable "org_uuid" {
+  description = "The CircleCI organization UUID that will be authorized to assume the role."
+  type        = string
+  nullable    = false
```

Comment:
When `create = false`, the variables without default values must still be set since nullable is set to `false`. Right?

https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/examples/iam-github-oidc/main.tf#L57-L61

---

Please add examples for this module similar to https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/examples/iam-github-oidc/

Reply:
Yes, thinking about it, what is the point in the `create` variable?
Will add the example.